### PR TITLE
Add VARIDESK.app latest

### PR DESCRIPTION
**Pull request by proinsias**
Add the VARIDESK Standing Desk Companion App that reminds you to stand
up occasionally.

### CODE REVIEW

File: Casks/varidesk.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'varidesk' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.varidesk.com/assets/standing-desk-app/VARIDESK.dmg'
+  name 'VARIDESK Standing Desk Companion App'
+  name 'VARIDESK'
```
Comment (vitorgalvao) [inline Casks/varidesk.rb]: This line must go.

**Comment by vitorgalvao** [comment]
It’s missing `uninstall`. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) geared for that.

**Comment by proinsias** [comment]
Ah! First time using "installer manual:". I think I've fixed it.

**Comment by adidalal** [comment]
@vitorgalvao Your TextExpander snippets need updating, I think.

**Comment by adidalal** [comment]
Thank you for the contribution. It was merged directly as 3f6d30cc0a9c8ef0dcb91f76720b00326bd48e09 to keep commit history cleaner. Your contribution is still credited to you.

**Comment by vitorgalvao** [comment]
@adidalal Thank you, will update.

**Pull request closed without merging**